<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateOpeningBalance</class>
 <widget class="QDialog" name="UpdateOpeningBalance">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>573</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Expense Tracker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/expense_tracker_sm.png</normaloff>:/icons/expense_tracker_sm.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(245, 237, 193);
}

#labelMessage{
color: rgb(255, 0, 0);
}


QLineEdit{
		color: rgb(0, 0, 255);
        border: 1px solid #264BF6;
}

#editBalance{
	background-color: rgb(245, 245, 245);
	color: rgb(0, 0, 0);

}

QPushButton{
      background-color : rgb(255, 255, 182);
      border: 1px solid #264BF6;
}                    

QPushButton::hover{
	 background-color : rgb(0, 0, 255); 
	 color : rgb(255, 255, 255);
}

QPushButton::disabled{
	background-color : rgb(0, 0, 0); 
	color : rgb(255, 255, 255);

}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QFrame" name="frameBody">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>25</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="labelBankName">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bank Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editBankName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>40</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelOpeningBalance">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Opening Balance</string>
        </property>
        <property name="buddy">
         <cstring>editBalance</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editBranchName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>40</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editCustomerName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>40</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCustomerName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Customer Name</string>
        </property>
        <property name="buddy">
         <cstring>editCustomerName</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="editBalance">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(245, 245, 245);</string>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelAccountNumber">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Account Number</string>
        </property>
        <property name="buddy">
         <cstring>editAccountNumber</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelBranchName">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Branch Name</string>
        </property>
        <property name="buddy">
         <cstring>editBranchName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editAccountNumber">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="maxLength">
         <number>25</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameFooter">
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBack">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&amp;BACK</string>
        </property>
        <property name="icon">
         <iconset resource="../src/resource.qrc">
          <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOk">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&amp;OK</string>
        </property>
        <property name="icon">
         <iconset resource="../src/resource.qrc">
          <normaloff>:/icons/icons/ok.png</normaloff>:/icons/icons/ok.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editAccountNumber</tabstop>
  <tabstop>editCustomerName</tabstop>
  <tabstop>editBranchName</tabstop>
 </tabstops>
 <resources>
  <include location="../src/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>UpdateOpeningBalance</receiver>
   <slot>onOkPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBack</sender>
   <signal>clicked()</signal>
   <receiver>UpdateOpeningBalance</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editBalance</sender>
   <signal>textEdited(QString)</signal>
   <receiver>UpdateOpeningBalance</receiver>
   <slot>onTextEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAccountChanged()</slot>
  <slot>onCancelPressed()</slot>
  <slot>onOkPressed()</slot>
  <slot>onTextEdited()</slot>
  <slot>onBankNameChanged()</slot>
 </slots>
</ui>
